beta-src/src/state/game/game-api-slice.ts

```diff
@@ -227,6 +228,16 @@ const gameApiSlice = createSlice({
       } = clickData;
       if (order.inProgress) {
         const currOrderUnitID = order.unitID;
+        let contextVariables;
+        let parsedContext;
+        let gamePhase;
+        if ("contextVars" in gameData) {
+          const { contextVars } = gameData;
+          contextVariables = contextVars;
+          parsedContext = JSON.parse(contextVariables.context);
+          const { phase } = parsedContext;
+          gamePhase = phase;
+        }
```

Comment:
why are you getting the phase from the context? The phase is accessible from the `overview` state. I recommend you do a 30 minute read through of the data in the global state.